coc/war_clans.py

```diff
@@ -147,6 +147,15 @@ def defenses(self) -> typing.List["WarAttack"]:
         """
         other = self._war and self._war.clan if self.is_opponent else self._war.opponent
         return other.attacks if other else []
+    
+    @property
+    def average_attack_duration(self) -> int:
+        """:class:`int`: Returns the average duration of all clan member's attacks this war."""
+        total_duration = 0
+        count = len(self._cs_attacks)
+        for attack in self._cs_attacks:
```

Comment:
The [sum](https://docs.python.org/3/library/functions.html#sum) function can do this more quickly and efficiently, something like `total_duration = sum(attack.duration for attack in self.attacks)`